packages/site/pages/install.js

```diff
@@ -69,8 +69,8 @@ module.exports = {
 `}</Doc>
       <P>
         <Badge color={Badge.colors.yellow}>Important</Badge> See the{' '}
-        <TextLink href="/core/build">CSS Build docs</TextLink> for other CSS
-        tech config options if the above doesn't apply.
+        <TextLink href="/config">CSS config docs</TextLink> for other CSS tech
```

Comment:
What was the motivation for the url change?

Reply:
Moving CSS configs up in nav hierarchy as dev questions about setup often relate to this. Right now those configs are only available through a link on install page rather than being at the same nav level as install in the sidebar. Talked to @danethurber  about it and he concurred.

Reply:
Moving CSS configs up in nav hierarchy as dev questions about setup often relate to this. Right now those configs are only available through a link on install page rather than being at the same nav level as install in the sidebar. Talked to @danethurber  about it and he concurred.